lib/permutations.rb

```diff
@@ -1,4 +1,22 @@
-
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
-end
\ No newline at end of file
+  hash = {}
+  string_permutation = true
+
+  if string1.length != string2.length
+    string_permutation = false
+  else
+      string1.each_char do |character|
+        hash.store(character, true)
+      end
+      
+      string2.each_char do |character|
+        if hash[character] == nil
+          string_permutation = false
+        end
```

Comment:
This unfortunately doesn't take into account the number of each character.  

I'll add another test for C13 when they get this assignment.